tools/java/org/unicode/cldr/util/Emoji.java

```diff
@@ -92,6 +98,9 @@
             }
             Iterator<String> it = semi.split(line).iterator();
             String emojiHex = it.next();
+            if (emojiHex.contains("1F6FC")) {
+                int debug = 0;
+            }
```

Comment:
The above 3 lines seem kind of useless, if harmless. Set block-local debug to 0 if current emoji is ROLLER SKATE?

Reply:
That's debugging code. (setting a conditional break point makes it exceedingly slow). There are a number of these in the code that could be removed as we wish.